app/views/topical_events/show.html.erb

```diff
@@ -12,12 +12,13 @@
       context: (I18n.t("topical_events.archived") if @topical_event.archived?)
     } %>
 
-    <%= render "govuk_publishing_components/components/lead_paragraph", {
-      text: @topical_event.description,
-      margin_bottom: 8,
-    } %>
+    <div class="govuk-!-margin-bottom-8">
+      <%= render "govuk_publishing_components/components/lead_paragraph", {
+        text: @topical_event.description,
+      } %>
 
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```

Comment:
I wonder if rather than needing an extra div for the margin, whether we can just apply it to this hr?

I imagine it's worth explaining in the commit that we want the paragraph close to the HR and then apply the vertical margin.